src/clj/rems/application/commands.clj

```diff
@@ -587,8 +619,8 @@
   (-> event
       (update :application/id (fn [app-id]
                                 (or app-id (:application-id cmd))))
-      (assoc :event/time (:time cmd)
-             :event/actor (:actor cmd))))
+      (assoc :event/time (:time cmd))
+      (update :event/actor #(or % (:actor cmd)))))
```

Comment:
this is the first command that needs to emit events with a different actor

Reply:
https://github.com/CSCfi/rems/pull/2415/files#diff-610efaa529ccf78673019d8914aa204c63a43d5ca18eb1d8f8f94dfb57398be5R530

Reply:
But actor of the invitation is the inviter and actor of the join is the person joining?

Reply:
it's not needed for the current implementation any more, but it's a nice feature, letting the command handler override the actor if needed